### ISSUE TITLE
Is there any way to pass async option to XML2JS

### ISSUE THREAD

**Issue by pj-esportudo** [bobby-brennan/rss-parser#44]
I am getting the following error: 
i tried to fix it by passing async : true and it worked. is there any workaround for this issue?
Code:
D:\Backend\node_modules\xml2js\lib\parser.js:329
          throw err;
          ^

Error: Text data outside of root node.
Line: 1
Column: 1
Char: ﻿
    at error (D:\Backend\node_modules\sax\lib\sax.js:651:10)
    at strictFail (D:\Backend\node_modules\sax\lib\sax.js:677:7)
    at Object.write (D:\Backend\node_modules\sax\lib\sax.js:1035:15)
    at Parser.exports.Parser.Parser.parseString (D:\Backend\node_modules\xml2js\lib\parser.js:322:31)
    at Parser.parseString (D:\Backend\node_modules\xml2js\lib\parser.js:5:59)
    at Object.exports.parseString (D:\Backend\node_modules\xml2js\lib\parser.js:354:19)
    at Object.Parser.parseString (D:\Backend\node_modules\rss-parser\index.js:250:10)
    at IncomingMessage.<anonymous> (D:\Backend\node_modules\rss-parser\index.js:297:21)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)

**Comment by bobby-brennan** [comment]
I think a good way to do this would be to have a passthrough for xml2js options, e.g.

```js
var options = {
  xml2js: {async: true},
}
rssParser.parseUrl('http://example.com/rss', options, function(err, data) {
  console.log(err, data);
});
```
where `options.xml2js` gets passed to `XML2JS.parseString()`.

Feel free to send a PR, otherwise I will take a look when I have some time.

**Comment by pj-esportudo** [comment]
I tried this and its still not working. I looked into` index.js` and parameters are not getting passed

Line 255: 
```
Parser.parseString = function(xml, options, callback) {
  if (!callback) {
    callback = options;
    options = {};
  }
  XML2JS.parseString(xml, function(err, result) {

```

**Comment by bobby-brennan** [comment]
Sorry for the confusion, I was suggesting you edit the code to pass the options through. But I'm happy to implement...if you're able to share the feed that triggered the error above, we can add a test case as well.

**Comment by Esportudo** [comment]
Most of the feeds that I use keep updating so it might break today but will work fine tomorrow. 
here are few examples:
```
http://torcedores.uol.com.br/equipes/sao-paulo/feed
http://m.goal.com/s/br/team/1
```

**Comment by bobby-brennan** [comment]
Sorry for the delay, this is implemented in 2.11.0

**Comment by pj-esportudo** [comment]
Thanks for the update.
